gherkin-utils/java/gherkin-utils/src/main/java/gherkin/utils/formatter/GherkinPrettyFormatter.java

```diff
@@ -0,0 +1,101 @@
+package gherkin.utils.formatter;
+
+import gherkin.utils.formatter.model.BackgroundStepContainer;
+import gherkin.utils.formatter.model.IStepContainer;
+import gherkin.utils.formatter.model.ScenarioStepContainer;
+import io.cucumber.messages.Messages.GherkinDocument;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class GherkinPrettyFormatter {
```

Comment:
All classes that are part of the API should be final.

Reply:
Sr, this is my mistake. I have already fixed.